src/app/pages/BuyPage/components/Tabbar/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+import { colors } from '../../../../constants';
+
+interface ActiveProps {
+  setSelectedTab: (e: boolean) => void;
+  selectedTab: boolean;
+}
+
+const TabContainer = styled.div`
```

Comment:
Please convert the `styled-components` into `module css`.

Reply:
fixed